modules/nvidia-gpu-architecture.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/about-hardware-accelerators.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nvidia-gpu-architecture_{context}"]
+= NVIDIA GPU architecture
+
+NVIDIA supports the use of graphics processing unit (GPU) resources on {product-title}. {product-title} is a security-focused and hardened Kubernetes platform developed and supported by Red Hat for deploying and managing Kubernetes clusters at scale. {product-title} includes enhancements to Kubernetes so that users can easily configure and use NVIDIA GPU resources to accelerate workloads.
```

Comment:
Same comment as above.

Reply:
No change on this one. The wording was chosen by devs because of unique relationship between OpenShift and partner accelerators.